python/doc/examples/calibration/calibration_logistic.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Calibration of the logistic model"
+    "# Calibration of the logistic model\n",
+    "\n",
+    "In this example, we calibrate the parameters of a model which predicts the dynamics of the size of a population depending on the time in a given time interval. This shows how a model which predicts a time dependent output can be calibrated by managing the dynamics as a vector output of the model."
```

Comment:
The first sentence seems redundant to me, because the word "dynamics" implies the model is time-dependant.
The second is too complicated in my opinion.
```suggestion
    "In this example, we calibrate the parameters of a model which predicts the dynamics of the size of a population. This shows how you can calibrate a model which predicts a time dependent output. You need to view the output time series as a vector."
```